test/README.md

```diff
@@ -17,13 +17,29 @@ a list of cells to ignore for each notebook, which is useful when specific cells
 intentionally erroneous. Each notebook is then loaded in a browser page, and all its cells are
 executed, and tests pass if no errors are produced.
 
-Tests were written as a nodejs package, so in order to run them locally, you can do `npm install`
-then `npm test` (or `npm install-test`). The one dependency is you have to be able to install
+## Running Tests
+
+### TL;DR
+
+Make sure you can install cairo on your system. Check the `.travis.yml` file for more info
```

Comment:
Well the installation is different according to your platform and set up, I can't list all steps here. If I list them for Ubuntu for example, it's different from one version to the other, and your specific machine might be missing some packages. I don't want to put a list of steps here that will only work for a subset of developers.